charts/opa-kube-mgmt/values.yaml

```diff
@@ -134,7 +134,7 @@ mgmt:
   enabled: true
   image:
     repository: openpolicyagent/kube-mgmt
-    tag: null # appVersion is used by default, set to desired value to override
+    tag: "0.0.0" # appVersion is used by default, set to desired value to override
```

Comment:
if you change tag to `0.0.0` then here should change either, https://github.com/open-policy-agent/kube-mgmt/blob/305bd71da00d17b1f379bf7d4636ef2af9179f5b/charts/opa-kube-mgmt/templates/_helpers.tpl#L106

Reply:
@eshepelyuk

Reply:
thanks! will fix it.